assets/js/youtube.js

```diff
@@ -1,14 +1,14 @@
-var apiKey = "AIzaSyD3d4qZsAk3Dz7EfLV3o227q1o6gY2MKmA";
+var apiKey = "AIzaSyCV6PrVjxbthsHbXRdvNJ2UorJa4NKm3R0";
 
 function getYoutubeData(searchQuery) {
   fetch(
-    `https://www.googleapis.com/youtube/v3/search?part=snippet&q=${searchQuery}&maxResults=10&type=video&videoSyndicated=true&videoLicense=creativeCommon&key=${apiKey}`
+    `https://www.googleapis.com/youtube/v3/search?part=snippet&q=${searchQuery}&maxResults=11&type=video&videoSyndicated=true&videoLicense=creativeCommon&key=${apiKey}`
   )
     .then((response) => response.json())
     .then((data) => {
       var videoInfo = data.items;
       var youtubeVideo = document.getElementById("videoContainer");
-      for (var i = 0; i < data.items.length; i++) {
+      for (var i = 1; i < data.items.length; i++) {
```

Comment:
i took out the first item out of the array because youtube is returning a video that we cannot  show on our site.  You can see this reflected in starting at index 1 and getting 11 results instead of 10.